DEVELOPER.md

```diff
@@ -23,3 +23,14 @@ Run the browser tests using Karma:
 Run the node.js tests:
 
 `npm run test-node`
+
+Before Commit
+------------
+
+Pre-commit hook will be executed and following check will
+automatically run before commit.
+
+- tslint
```

Comment:
Yeah, test in monitor mode can make sure all jasmine test passed.
But with pre commit, the hook can check format and lint.

Are you suggesting when code changes, automatically run format and lint check?